components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIImportUtil.java

```diff
@@ -531,8 +532,13 @@ private static void updateAPIWithThumbnail(File imageFile, API importedApi, APIP
                 // Check whether the icon is in .json format (UI icons are stored as .json)
                 new JsonParser().parse(new FileReader(imageFile));
                 mimeType = APIConstants.APPLICATION_JSON_MEDIA_TYPE;
-            } catch (Exception e) {
-                log.error("Failed to read the file. ", e);
+            } catch (JsonParseException e) {
```

Comment:
Shall we use `} catch (JsonParseException | FileNotFoundException e) {`

Reply:
I have used two exceptions to display two different logs.